deploy/addon/pulsar.md

```diff
@@ -0,0 +1,259 @@
+---
+title: Pulsar
+shortdesc: A distributed messaging and streaming platform on the publish/subsribe model
+tags:
+- addons
+keywords:
+- pulsar
+- messaging
+- cold storage
+- queue
+- hot storage
+- biscuit
+---
+
+## Overview
+
+Pulsar works on a publisher / subscriber model, just like RabbitMQ or Kafka.
+
+```
+client               Pulsar             client
+publishes to   =>    topic      <=  subscribes to
+
+"producer"                           "consumer"
+```
+
+They are several modes of subscription. A consumer may subscribe exclusively, or share the subscription with other consumers.
+
+- Exclusive
+- Failover (if a consumer fails, another one receives the message)
+- Shared (messages are distributed to several consumers)
+- Key_Shared (messages come with keys and go to consumers with the corresponding key)
+
+More on this in the [official documentation](https://pulsar.apache.org/docs/en/concepts-messaging/#subscriptions)
+
+A topic is defined this way:
+
+`{persistent|non-persistent}://tenant/namespace/topic`
+
+Tenants and namespaces allow for grouping and subgrouping of topics.
+
+What a pulsar add-on basically is, is a tenant/namespace pair that allows you to create and use as many topics as you wantt following this pattern:
+
+`{persistent|non-persistent}://user_id/pulsar_addon_id/name_of_your_topic`
+
+## Authorization
+
+Pulsar addon uses (Biscuit for Pulsar)[https://github.com/CleverCloud/biscuit-pulsar/commits/master] implementation which is directly pluggable to the Pulsar authentication and authorization system. Each addon exposes its own biscuit token.
+
+### Usage
+
+We advise you to use (`pulsarctl`)[https://github.com/streamnative/pulsarctl] provided by StreamNative. Here is an example to list topics in your addon (in your namespace):
+
+```bash
+pulsarctl --admin-service-url https://<PULSAR_REST_URL> --auth-params <BISCUIT> --auth-plugin org.apache.pulsar.client.impl.auth.AuthenticationToken topics list <tenant>/<namespace>
+```
+
+As Biscuit is a token, you can use `AuthenticationToken("<BISCUIT>")` provided by clients librairies to authenticate to our clusters without any tweak.
+
+* C++ client
```

Comment:
I think should simply link this page: https://pulsar.apache.org/docs/en/client-libraries/